pass/cfg.go

```diff
@@ -17,7 +17,7 @@ func LabelTarget(fn *ir.Function) error {
 		switch n := node.(type) {
 		case ir.Label:
 			if pending != empty {
-				return errors.New("instruction should follow a label")
+				return fmt.Errorf("instruction should follow a label %q", pending)
```

Comment:
The grammar in this error message is now a little odd in my opinion. Just deleting the `a` would be better?

`instruction should follow label %q`

or

`expected instruction following label %q`